hocon/src/main/java/space/arim/dazzleconf/ext/hocon/HoconOptions.java

```diff
@@ -0,0 +1,102 @@
+package space.arim.dazzleconf.ext.hocon;
+
+import com.typesafe.config.ConfigParseOptions;
+import com.typesafe.config.ConfigRenderOptions;
+
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+
+public class HoconOptions {
+  private final ConfigParseOptions configParseOptions;
+  private final ConfigRenderOptions configRenderOptions;
+  private final Charset charset;
+
+  HoconOptions(Builder builder) {
+    this.configParseOptions = builder.configParseOptions;
+    this.configRenderOptions = builder.configRenderOptions;
+    this.charset = builder.charset;
+  }
+
+  /**
+   * Gets used parse options
+   *
+   * @return used parse options
+   */
+  public ConfigParseOptions configParseOptions() {
+    return configParseOptions;
+  }
+
+  /**
+   * Gets used render options.
+   *
+   * @return used render options
+   */
+  public ConfigRenderOptions configRenderOptions() {
+    return configRenderOptions;
+  }
+
+  /**
+   * Gets used charset
+   *
+   * @return used charset
+   */
+  public Charset charset() {
+    return charset;
+  }
+
+  /**
+   * Builder of {@code HoconOptions}
+   *
+   * @author CDFN
+   */
+  public static class Builder {
+    ConfigParseOptions configParseOptions = ConfigParseOptions.defaults();
+    ConfigRenderOptions configRenderOptions = ConfigRenderOptions.defaults()
+        .setOriginComments(false)
+        .setJson(false);
+
+    Charset charset = StandardCharsets.UTF_8;
+
+    /**
+     * Sets parse options. Reading config uses them.
+     *
+     * @param options the options
+     * @return this builder
+     */
+    public Builder configParseOptions(ConfigParseOptions options) {
+      this.configParseOptions = options;
```

Comment:
This method, and the other builder methods, should use Objects.requireNonNull to validate their arguments.